models/zeroex/ethereum/zeroex_api_ethereum_schema.yml

```diff
@@ -0,0 +1,65 @@
+version: 2
+
+models:
+  - name: zeroex_api_ethereum_fills
+    meta:
+      blockchain: ethereum
+      project: zeroex
+      contributors: sui414, bakabhai993
+    config:
+      tags: ['ethereum','0x','dex_aggregator','dex','aggregator']
+    description: >
+        0x API erc20 swaps raw fills (including multihops) and also native swaps thru 0x exchange contracts (without using 0x API) on Ethereum
+    columns:
+
+      - &tx_hash
+        name: tx_hash
+        description: "Transaction hash of the fill"
+      - &evt_index
+        name: evt_index
+        description: "Index of the corresponding fill/swap event"
+      - &contract_address
+        name: contract_address
+        desctiption: "The address of the contract which fired the fill/swap event"
+      - &block_time
+        name: block_time
+        description: "Timestamp for block event time in UTC"
+      - &block_date
+        name: block_date
+        description: "UTC event block date of each trade"
+      - &maker
+        name: maker
+        description: "For limit order, the wallet who posted order first; for AMM orders, the liquidity pools; for RFQ orders, the professional market makers."
+      - &taker
+        name: taker
+        description: "User who is taking the order"
+      - &taker_token
+        name: taker_token
+        description: "Contract address of the token taker sells"
+      - &maker_token
+        name: maker_token
+        description: "Contract address of the token taker buys"
```

Comment:
After the test seeds added, you can test the column value by adding lines:
```
      - &maker_token
        name: maker_token
        description: "Contract address of the token taker buys"
        tests:
          - dex_trades_token_bought:
              dex_trades_seed: ref('zeroex_ethereum_transactions')
```

Reply:
i don't think it is necessary